main/autogen/cache.cc

```diff
@@ -74,7 +86,7 @@ bool AutogenCache::canSkipAutogen(core::GlobalState &gs, string_view cachePath,
 // this doesn't load the whole cache, since we never need to _consult_
 // the whole cache. Instead, we let it know which paths we care about
 // and load only those
-AutogenCache AutogenCache::unpackForFiles(string_view file_contents, const UnorderedSet<string> &changedFiles) {
+AutogenCache AutogenCache::unpackForFiles(string_view file_contents, const UnorderedSet<string_view> &changedFiles) {
```

Comment:
Are you sure these refs live long enough? I haven’t checked the lifetimes fully but it’s frequently rare that containers like this want string_view over just string

Reply:
These refs should definitely live long enough: the set in question will never live longer than the vector that contains the strings they're built from.